common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -436,6 +436,46 @@ def test_get_items(self, default_ms, max_find, max_send):
                 revision=ModuleStoreEnum.RevisionOption.draft_preferred
             )
 
+    @ddt.data(ModuleStoreEnum.Type.split, ModuleStoreEnum.Type.mongo)
+    def test_get_items_in_tree_only(self, default_ms):
+        """
+        Test `in_tree_only` option helps in returning only those items which are present in course tree.
+        """
+        self.initdb(default_ms)
+
+        test_course = self.store.create_course('testx', 'GreekHero', 'test_run', self.user_id)
+        course_key = test_course.id
+
+        # Check items found in start is 2
+        items = self.store.get_items(course_key)
+        self.assertEqual(len(items), 2)
```

Comment:
assert the types of the blocks found in `items`. Assert that the about block has that "detached" tag.